wrapping/itkStripTsImageFilter.wrap

```diff
@@ -1,3 +1,8 @@
-itk_wrap_class("itk::SkullStrip" POINTER)
-  itk_wrap_image_filter("${WRAP_ITK_REAL}" 2 2+)
+itk_wrap_class("itk::StripTsImageFilter" POINTER)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    foreach(t ${WRAP_ITK_REAL})
+      itk_wrap_template("${ITKM_${t}}${d}${ITKM_${t}}${d}${ITKM_${t}}${d}"
+                        "itk::Image< ${ITKT_${t}},${d} >, itk::Image< ${ITKT_${t}}, ${d} >, "itk::Image< ${ITKT_${t}}, ${d} >")
```

Comment:
You have 3 `"` symbols on this line, meaning they are not matched. You probably need to remove the middle one.

Reply:
Good catch :eagle: @dzenanz ! Thanks. Let's see if e8d3485 makes it !